### ISSUE TITLE
[9] A TRANSFORMER-BASED FRAMEWORK FOR MULTIVARIATE TIME SERIES REPRESENTATION LEARNING

### ISSUE THREAD

**Issue by techzzt** [techzzt/pytorch-paper-implement#9]
> **A TRANSFORMER-BASED FRAMEWORK FOR MULTIVARIATE TIME SERIES REPRESENTATION LEARNING** 

_Zerveas, George, et al. "A transformer-based framework for multivariate time series representation learning." Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining. 2021._

![image](https://user-images.githubusercontent.com/49440852/182013385-76372e34-e555-4f49-a1ea-b9b65541ace0.png)

- 다변량 시계열 데이터의 unsupervised representation learning에 Transformer를 접목한 알고리즘 제안 
- Pre-training: unlabeled time series 데이터를 기반으로 masked input을 예측하는 task 수행
   -  binary noise mask: markov chain의 transition probability를 기반으로 mask segmentation 적용 
   - 연속적인 구간에 segmentation을 적용해 예측해가며 representation 최대화
- Fine-tuning: pre-train 모델의 가중치는 고정하고 labeled time series 분류 문제 수행
